Validation/MtdValidation/plugins/EtlLocalRecoValidation.cc

```diff
@@ -267,7 +574,8 @@ void EtlLocalRecoValidation::fillDescriptions(edm::ConfigurationDescriptions& de
   desc.add<std::string>("folder", "MTD/ETL/LocalReco");
   desc.add<edm::InputTag>("recHitsTag", edm::InputTag("mtdRecHits", "FTLEndcap"));
   desc.add<edm::InputTag>("recCluTag", edm::InputTag("mtdClusters", "FTLEndcap"));
-  desc.add<double>("hitMinimumEnergy", 1.);  // [MeV]
+  desc.add<double>("hitMinimumEnergy1Dis", 0.01);   // [MeV]
```

Comment:
@gsorrentino18 this threshold has changed compared to the default version also fo the old scenarios, why?